rental.rb

```diff
@@ -0,0 +1,20 @@
+require 'time'
+
+class Rental
+  attr_reader :person, :book
```

Comment:
- [ ] Please make the `person` and `book` accessors so you can set values to them later on
```ruby
attr_accessor :person, :book
```